docs/technical_reference/unit_models/ion_exchange_0D.rst

```diff
@@ -455,10 +455,11 @@ disposal costs are calculated as a function of the annual mass of resin replaced
 Where :math:`M_{res}` is the resin mass for a single bed and :math:`\dot{v}_{regen}` is the volumetric flow of regenerant solution. If ``hazardous_waste`` is set to ``False``,
 :math:`C_{op,haz} = 0`
 
-The total energy consumed by the unit is the summation of the power required for each of the booster pump, backwashing pump, regeneration pump, and rinsing pump:
+The total energy consumed by the unit is the summation of the power required for each of the booster pump, backwashing pump, regeneration pump, and rinsing pump. Each is scaled 
```

Comment:
One extremely subtle thing I noticed was in your pump power equations, you essentially are multiplying rho*g*h with Q divided by efficiency. However, instead of using head directly (`h`), you have some constant (0.7ish) multiplied by pressure drop in psi, leading me to believe your constant (assuming it's a conversion factor) multiplied by pressure drop equates to head added by the pump. 

Not necessarily anything wrong here--I just wanted to make sure I am interpreting correctly and looking for a little clarification.

Reply:
You are correct it is a conversion factor, was used before you can convert the units directly to kW. 

The conversion is not used anymore in the model equations but are still in the docs. I will fix- thanks for catching that.